cfgov/unprocessed/apps/paying-for-college/css/guides.css

```diff
@@ -503,7 +496,7 @@ This just the hero for the Comparison Tool.
 .breadcrumbs a {
     background-color: transparent;
     border-bottom: none;
-    color: #328ed8;
+    /* color: #328ed8; */
```

Comment:
Maybe just delete this?

Reply:
D'oh, meant to after testing it. Thanks.